doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -467,6 +467,34 @@ destroy all backtrack and return values.
 Backtracking
 ++++++++++++
 
+Earlier, we said that "the `Proofview.tactic` monad is essentially an
+IO monad together with backtracking state representing the proof
+state." What this means is that all Ltac2 expressions of type `'a`
+either fail to evaluate to a value (and raise an exception) or
+implicitly return, not just a value of type `'a`, but also an
```

Comment:
@ppedrot I hope you don't mind, I will ping you in a few comments here to get some fine-grained feedback on choices of wording.
> "all Ltac2 expressions of type `'a` either fail to evaluate to a value (and raise an exception) or implicitly return, not just a value of type `'a`, but also an exception handler which will be used if a computation "downstream" raises an exception."

This language is supposed to suggest that the "exception handler" does live past the successful evaluation of the current expression, i.e., it is a continuation.